### PR TITLE
Refactor CAT II Tasks

### PR DESCRIPTION
**Pull request by samdoran**
Refactor `cat2.yml` to use multi-line YAML and adhere to [coding standards](https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md).

See issue #29 for details.

### CODE REVIEW

File: tasks/cat2.yml [hunk 1]
```diff
@@ -1389,10 +1400,10 @@
       - audit
```
Comment (jtyr) [inline tasks/cat2.yml]: Wrong tag. It should be `patch` instead of `audit`.

**Comment by jtyr** [comment]
Please squash all the commits into one (`git rebase -i ...`) and update your PR (`git push -f`).

**Comment by defionscode** [comment]
Squashing isn't needed anymore, GItHub facilitates squashing at merge time
now.


[image: photo] <http://linkedin.com/in/davilaio>
<http://twitter.com/defionscode>  <http://github.com/defionscode> Jonathan
I. Davila
Principal Architect, Automation Practice, Red Hat, Inc
202.415.1878 // redhat.com // jdavila@redhat.com // Washington, D.C

**Comment by samdoran** [comment]
@jtyr: I corrected the tag. Thanks for pointing it out!

**Pull request merged**